libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
Somehow it must be a mistake if a rethrow is executed outside of a fault handler, hence, if possible, the developer should be notified at compile time.

I think only throwing a RuntimeError at runtime is dangerous as the developer could find out too late about his mistake. The same could be said about just executing a no-op: this could result in the developer never discovering his fault and thereby risking that the service is put into an unwanted state.